### PR TITLE
[SC-207] update cfn-cr-synapse-tagger version

### PR DESCRIPTION
**Pull request by zaro0508**

### CODE REVIEW

File: config/develop/cfn-cr-synapse-tagger.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@ stack_tags:
   OwnerEmail: "it@sagebase.org"
 hooks:
   before_launch:
-    - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/cfn-cr-synapse-tagger/0.0.3/cfn-cr-synapse-tagger.yaml --create-dirs -o templates/remote/cfn-cr-synapse-tagger.yaml"
+    - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/cfn-cr-synapse-tagger/0.0.4/cfn-cr-synapse-tagger.yaml --create-dirs -o templates/remote/cfn-cr-synapse-tagger.yaml"
```
Comment (brucehoff) [inline config/develop/cfn-cr-synapse-tagger.yaml:9]: Dumb question:  Where is the version "0.0.4" defined? I couldn't find it in the cfn-cr-synapse-tagger repo': https://github.com/Sage-Bionetworks-IT/cfn-cr-synapse-tagger/search?q=0.0.4

Comment (zaro0508) [reply config/develop/cfn-cr-synapse-tagger.yaml:9]: it's in our cloudformation repo, https://bootstrap-awss3cloudformationbucket-19qromfd235z9.s3.amazonaws.com/index.html#cfn-cr-synapse-tagger/0.0.4/

Comment (brucehoff) [reply config/develop/cfn-cr-synapse-tagger.yaml:9]: I see that's where the version can be _found_ but I was wondering where it is _defined_.  I see there is a tag "0.0.4" in the cfn-cr-synapse-tagger repo and also that the version appears here:  https://github.com/Sage-Bionetworks-IT/cfn-cr-synapse-tagger/blob/0.0.4/template.yaml#L18. Does the Travis-CI script (https://github.com/Sage-Bionetworks-IT/cfn-cr-synapse-tagger/blob/master/.travis.yml) create the tag, or is it that when you are ready to deploy you create the tag triggering Travis-CI to build and deploy?

Comment (zaro0508) [reply config/develop/cfn-cr-synapse-tagger.yaml:9]: > Does the Travis-CI script (https://github.com/Sage-Bionetworks-IT/cfn-cr-synapse-tagger/blob/master/.travis.yml) create the tag, or is it that when you are ready to deploy you create the tag triggering Travis-CI to build and deploy?

Tag is manually created then travis takes care of building and deploying that tag to our CFN artifacts repository. The tag is here:   https://github.com/Sage-Bionetworks-IT/cfn-cr-synapse-tagger/releases/tag/0.0.4

Comment (zaro0508) [reply config/develop/cfn-cr-synapse-tagger.yaml:9]: yeah we should really create tags for all those and pin scipool-infra to use specific versions.  would you be interested in working on that?

**Review by tthyer — APPROVED** [review]

**Pull request merged**